src/components/ZclDomainClusterView.vue

```diff
@@ -163,15 +163,15 @@ limitations under the License.
             }}
           </q-td>
           <q-td key="enable" :props="props">
-            <q-select
+            <q-option-group
```

Comment:
Is adding cypress tests to check for file open and make sure UI is responding easy?